.github/workflows/validatePR.yml

```diff
@@ -1,12 +1,13 @@
 name: Validate PR
 on:
   pull_request:
+    types: [opened, reopened, edited]
```

Comment:
Just an FYI when I did this in the EGPT repo it put open PRs into a weird state.  I got past it by just edit each description and it fired off the GHA, but initially they were stuck waiting on validation.  I think it was just a one time issue.

Reply:
Okay! I will keep that in mind and Watch out for PRs opened after this one is merged.